cmd/svcat/binding/unbind_cmd.go

```diff
@@ -55,10 +54,10 @@ func NewUnbindCmd(cxt *command.Context) *cobra.Command {
 		RunE:    command.RunE(unbindCmd),
 	}
 	command.AddNamespaceFlags(cmd.Flags(), false)
-	cmd.Flags().StringVar(
-		&unbindCmd.bindingName,
+	cmd.Flags().StringArrayVar(
```

Comment:
FYI, there is a known bug in cobra around StringArray types. Switch this to StringSliceVar and everything looks the same but avoids the bug.

Reply:
Thanks. Fixed.